src/pages/ListUsers.tsx

```diff
@@ -53,9 +48,37 @@ const ListUsers = (): ReactElement => {
             });
     }, []);
 
+    const nonTestUsers = users?.filter((user) => !getAttributeValue(user, 'custom:noc')?.includes('IWBusCo'));
+
+    const completedRegisteredUsers = nonTestUsers.filter((user) => {
```

Comment:
I would use shorthand return syntax for these filters, eg. `const completedRegisteredUsers = nonTestUsers.filter((user) => user?.UserStatus === 'CONFIRMED')`